### PR TITLE
update rust and gh cli

### PR DESCRIPTION
**Pull request by jacob-delgado**

### CODE REVIEW

**Comment by ericvn** [comment]
crane 0.13.0 is another possible addition

**Review by howardjohn — APPROVED** [review]

**Comment by ericvn** [comment]
@jacob-delgado I see it is failing on:
```
The following contributors were found for this pull request:

❌ https://github.com/istio/tools/commit/8f64585c0660b0ea124c1346f18b438d5747ec42 Author: @jacob-delgado <jac*******ado​@volunteers.acasi.info>
```

File: docker/build-tools/Dockerfile [hunk 1]
```diff
@@ -42,7 +42,7 @@ ENV GO_JUNIT_REPORT_VERSION=df0ed838addb0fa189c4d76ad4657f6007a5811c
 ENV GOCOVMERGE_VERSION=b5bfa59ec0adc420475f97f89b58045c721d761c
 ENV GOIMPORTS_VERSION=v0.1.0
 ENV BENCHSTAT_VERSION=9c9101da8316
-ENV GH_VERSION=2.21.2
+ENV GH_VERSION=2.22.0
```
Comment (ericvn) [inline docker/build-tools/Dockerfile:45]: Also see a newer 2.22.1.

**Comment by jacob-delgado** [comment]
> @jacob-delgado I see it is failing on:
> 
> ```
> 
> The following contributors were found for this pull request:
> 
> 
> 
> ❌ https://github.com/istio/tools/commit/8f64585c0660b0ea124c1346f18b438d5747ec42 Author: @jacob-delgado <jac*******ado​@volunteers.acasi.info>
> 
> ```
> 
> 

Can you put up a PR in lieu of this one? I'm working with my company to understand what has changed. It may take several weeks.

**Pull request closed without merging**